x-pack/plugins/triggers_actions_ui/public/application/sections/alert_summary_widget/components/active_alert_counts.tsx

```diff
@@ -20,23 +27,26 @@ interface Props {
 
 export const ActiveAlertCounts = ({ activeAlertCount }: Props) => {
   const { euiTheme } = useEuiTheme();
+  const describedTextId = useGeneratedHtmlId();
 
   return (
     <>
-      <EuiText
-        color={!!activeAlertCount ? euiTheme.colors.dangerText : euiTheme.colors.successText}
-      >
-        <h3 data-test-subj={ACTIVE_ALERT_COUNT_DATA_TEST_SUBJ}>
+      <EuiTitle size="s">
+        <EuiTextColor
+          data-test-subj={ACTIVE_ALERT_COUNT_DATA_TEST_SUBJ}
+          color={!!activeAlertCount ? euiTheme.colors.dangerText : euiTheme.colors.successText}
+          aria-describedby={describedTextId}
```

Comment:
What is the usage of this field? When should we add it?

Reply:
@1Copenut, could you please double-check my thoughts? I was considering linking the number to the text related to that value. I believe aria-describedby would be ideal to accomplish this. Am I right?

Reply:
If you're using ARIA, could I suggest `aria-labelledby` with two IDs, one that references the number iteself and one that references the text? That way the focusable element's calculated accessible value is "1 alert" or "1 active now". I think `aria-describedby` will show up in the Forms / Links menus for screen readers as "1, more information available" or just "1", which isn't as meaningful.